1_fetch/src/data_selection.R

```diff
@@ -21,6 +21,10 @@ clean_monthly <- function(in_file)
     mutate(meets_criteria = n_per_month >= (days_in_month-2)) %>%
     filter(meets_criteria)
   
+  
+  dat <-  dat %>% 
+    mutate(n_years = length(unique(month_year))) %>% 
```

Comment:
I think this will calculate one value for the entire dataset, when I think you're looking to calculate this value per seg_id_nat - month.